src/controller/CHIPDeviceController.cpp

```diff
@@ -1628,7 +1628,6 @@ void DeviceCommissioner::OnNodeIdResolved(const chip::Dnssd::ResolvedNodeData &
 
     if (mDeviceBeingCommissioned != nullptr && mDeviceBeingCommissioned->GetDeviceId() == nodeData.mPeerId.GetNodeId())
     {
-        RendezvousCleanup(CHIP_NO_ERROR);
```

Comment:
So this RendezvousCleanup was removed because it would trigger duplicate pairing complete calls on the delegate. 
Note that this callback is only triggered for operational discovery, which means the actual Rendezvous is complete by now and so should have been cleaned up by EstablishPASE